src/index/vespa_.py

```diff
@@ -105,10 +111,31 @@ class VespaFamilyDocument(BaseModel):
     document_content_type: Optional[str] = None
     document_cdn_object: Optional[str] = None
     document_source_url: Optional[str] = None
+    document_title: Optional[str] = None
+    family_geographies: Optional[list[str]] = None
+    corpus_import_id: Optional[str] = None
+    corpus_type_name: Optional[str] = None
+    collection_title: Optional[str] = None
+    collection_summary: Optional[str] = None
+    metadata: list[MetadataItem]
+
+
+def reshape_metadata(
+    metadata: dict[str, list[str]]
+) -> list[VespaFamilyDocument.MetadataItem]:
+    metadata_items = []
+    for key, values in metadata.items():
+        metadata_items.extend(
+            [
+                VespaFamilyDocument.MetadataItem(name=key, value=value)
+                for value in values
+            ]
+        )
+    return metadata_items
```

Comment:
Related to the suggestion above. It would be `None` right, if there wasn't any metadata?
```suggestion
def reshape_metadata(
    metadata: Optional[dict[str, list[str]]]
) -> list[VespaFamilyDocument.MetadataItem]:
    if metadata is None:
        return None
        
    metadata_items = []
    for key, values in metadata.items():
        metadata_items.extend(
            [
                VespaFamilyDocument.MetadataItem(name=key, value=value)
                for value in values
            ]
        )
    return metadata_items
```

Reply:
👍